sdk/core/core-client/src/serviceClient.ts

```diff
@@ -128,10 +127,10 @@ export class ServiceClient {
    * @param {OperationArguments} operationArguments The arguments that the HTTP request's templated values will be populated from.
    * @param {OperationSpec} operationSpec The OperationSpec to use to populate the httpRequest.
    */
-  async sendOperationRequest(
+  async sendOperationRequest<T>(
```

Comment:
consider adding doc for type parameter? https://typedoc.org/guides/doccomments/#%40typeparam-%3Cparam-name%3E

Reply:
Huh, didn't know about this tag. Added.

Reply:
FYI: It is also supported by TSDoc: https://tsdoc.org/pages/tags/typeparam/